Disable musttail attribute, not supported by clang on powerpc, powerpc64
and mips64.

Index: Source/WTF/wtf/Compiler.h
--- Source/WTF/wtf/Compiler.h.orig
+++ Source/WTF/wtf/Compiler.h
@@ -321,7 +321,7 @@
 /* MUST_TAIL_CALL */
 
 #if !defined(MUST_TAIL_CALL) && defined(__cplusplus) && defined(__has_cpp_attribute)
-#if __has_cpp_attribute(clang::musttail)
+#if __has_cpp_attribute(clang::musttail) && !defined(__mips__) && !defined(__powerpc__)
 #define MUST_TAIL_CALL [[clang::musttail]]
 #endif
 #endif
